src/cleed++/Core/Basis.hh

```
@@ -0,0 +1,74 @@
+ */
+class Basis {
+  public:
+    Basis();
+    Basis(const basis_t *a);
+    Basis(const Basis &a);
+    ~Basis();
+
+    /* setters */
+    Basis& setA1(const Coordinate &a1);
+    Basis& setA1(double a1_x, double a1_y, double a1_z);
+    Basis& setA2(const Coordinate &a3);
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)